### PR TITLE
mkcloud/qa_crowbarsetup: Move backup/restore code to qa_crowbarsetup.sh

### PR DESCRIPTION
**Pull request by vuntz**
This is required so we can set REPOS_SKIP_CHECKS before restoring backup.

### CODE REVIEW

**Comment by vuntz** [comment]
Currently testing the change, but reviews welcome.

File: scripts/mkcloud [hunk 1]
```diff
@@ -1110,12 +1110,8 @@ function crowbarrestore()
         return 56
```
Comment (MaximilianMeister) [inline scripts/mkcloud]: as we're already here, maybe worth to complain instead of echo and return

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -2569,6 +2569,23 @@ function onadmin_cloudupgrade_reboot_and_redeploy_clients()
...
+{
+    AGREEUNSUPPORTED=1 CB_BACKUP_IGNOREWARNING=1 bash -x /usr/sbin/crowbar-backup backup /tmp/backup-crowbar.tar.gz
+}
+
+function onadmin_cloudrestore()
+{
+    AGREEUNSUPPORTED=1 CB_BACKUP_IGNOREWARNING=1 bash -x /usr/sbin/crowbar-backup purge
+
+    # Need to install crowbar-backup again as purge deletes it
+    zypper --non-interactive in --no-recommends crowbar
```
Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: i meant here `safely zypper` ... sorry, commented in your fork

File: scripts/mkcloud (hunk 1 above)
Comment (vuntz) [inline scripts/mkcloud]: done

File: scripts/qa_crowbarsetup.sh (hunk 2 above)
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: done

**Comment by vuntz** [comment]
I tested this, and this unbreaks backup/restore for Cloud 5.

**Comment by MaximilianMeister** [comment]
+1

**Comment by jdsn** [comment]
:+1:

**Pull request merged**